### ISSUE TITLE
student model need to finetune?

### ISSUE THREAD

**Issue by zhxjlbs** [anubhavashok/N2N#9]
Hi, Thank you for your advice. I get accuracy 92.01 of teacher model. And I still have a question. 
When training the students model in stage 1,there are many function to choose as I list below
1. def trainTeacherStudent()
2. def trainTeacherStudentRand()
3. def trainTeacherStudentNew()
4. def trainTeacherStudentParallel()
which one should I choose? 

I notice that these function hase different loss function. after training with function 4(def trainTeacherStudentParallel) in stage 1, the student model I got  is not as good as the student model in the paper.Whether finetune is required?

thank you!

**Comment by anubhavashok** [comment]
Yes, you need to finetune the model at the end to achieve the results in the paper. I recommend combining the soft targets with hard labels as mentioned in the paper by combining the losses with a lambda parameter (the optimal value depends on the particular student network, so do try a few)
